modules/get-started/pages/licensing/overview.adoc

```diff
@@ -31,27 +31,80 @@ The Enterprise Edition is licensed with the https://github.com/redpanda-data/red
 
 === License keys
 
-Enterprise features require an Enterprise Edition license key, sometimes referred to as Enterprise license, license key, or license. You can evaluate enterprise features with a free 30-day trial. To request a trial license key, to extend your trial period, or to purchase a license, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda Sales^].
+Enterprise features require an Enterprise Edition license key, sometimes referred to as Enterprise license, license key, or license.
+
+Starting with version 24.3, new Redpanda clusters automatically receive a trial license that's valid for 45 days, allowing unrestricted use of Enterprise features. This evaluation period begins when the cluster is created for the first time. After this period expires, inactive Enterprise features are disabled, and active features enter a restricted state.
+
+To request a trial license key, to extend your trial period, or to purchase a license, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda Sales^].
 
 [[self-managed]]
 === Enterprise features in Redpanda Self-Managed
 
-The following enterprise features for Redpanda Self-Managed are activated with a valid Enterprise Edition license key:
+The following table lists the available Enterprise features for Redpanda Self-Managed, along with the restrictions applied if the Enterprise license is invalid or has expired.
+
+[cols="1,2,2"]
+|===
+| Feature | Description | Restrictions if License Invalid
```

Comment:
Sorry one more nit:

Maybe instead of "License invalid" maybe "Restrictions if License has Expired" or "Restrictions if License Absent"?

Reply:
@michael-redpanda what about this? I think it covers both cases:

- Absent: The cluster has no license.
- Expired: Redpanda behaves as though no valid license exists. The license is no longer considered valid in this state, so the "without license" restrictions apply.

Reply:
Yes!